.github/workflows/rust.yml

```diff
@@ -0,0 +1,28 @@
+name: Rust
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup
+      run: |
+              sudo apt-get update -q
+              sudo apt-get -y install secilc
+    - name: Build
+      run: cargo build --verbose
```

Comment:
We should reuse [build.sh](https://github.com/dburgener/cascade/blob/main/.pipelines/build.sh) and similar scripts if possible, or at least their content (e.g. `rustup run stable cargo build --verbose`).